challenge_3/style.css

```
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  background: #62306D;
+  height: 300px;
+  position: absolute;
+  width: 400px;
+}
+
+.union1 {
+  background: #F7EC7D;
+  border-bottom-left-radius: 3em 3em;
+  border-bottom-right-radius: 3em 3em;
+  height: 105px;
+  margin-left: 50px;
+  margin-top: 150px;
```

Remember: `margin: top right bottom left` as in a clock🕐 . Change these to be only one: 
```suggestion
  margin: 150px 0 0 50px;
```